src/bin/cargo-plumbing/cli.rs

```diff
@@ -1,18 +1,21 @@
 use cargo_plumbing::CargoResult;
 use clap::Parser;
 
+use crate::plumbing::locate_project;
+
 #[derive(Debug, Parser)]
 #[command(bin_name = "cargo")]
 #[command(styles = clap_cargo::style::CLAP_STYLING)]
 pub(crate) enum Command {
-    #[command(subcommand)]
-    Plumbing(crate::plumbing::Plumbing),
```

Comment:
My plan had been for all plumbing commands to be namespaced under another command, `plumbing` being a placeholder for that.

Cargo also invokes plugins as `cargo-<cmd> <cmd>` so if we did this, we'd have to have a separate binary per plumbing command